rest-api/src/test/java/gov/cms/qpp/conversion/api/acceptance/CpcApiAcceptance.java

```diff
@@ -0,0 +1,189 @@
+package gov.cms.qpp.conversion.api.acceptance;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import gov.cms.qpp.conversion.api.helper.JwtPayloadHelper;
+import gov.cms.qpp.conversion.api.helper.JwtTestHelper;
+import gov.cms.qpp.conversion.api.model.CpcFileStatusUpdateRequest;
+
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.truth.Truth.assertThat;
+import static io.restassured.RestAssured.get;
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.put;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.xml.HasXPath.hasXPath;
+
+@ExtendWith(RestExtension.class)
+class CpcApiAcceptance {
+
+	private static final String CPC_UNPROCESSED_FILES_API_PATH = "/cpc/unprocessed-files";
+	private static final String CPC_FILE_API_PATH = "/cpc/file/";
+	private static final String PROGRAM_NAME_XPATH = "/*[local-name() = 'ClinicalDocument' and namespace-uri() = 'urn:hl7-org:v3']"
+		+ "/./*[local-name() = 'informationRecipient' and namespace-uri() = 'urn:hl7-org:v3']"
+		+ "/*[local-name() = 'intendedRecipient' and namespace-uri() = 'urn:hl7-org:v3']"
+		+ "/*[local-name() = 'id' and namespace-uri() = 'urn:hl7-org:v3'][@root='2.16.840.1.113883.3.249.7']/@extension";
+	private static final String CPC_PLUS_PROGRAM_NAME = "CPCPLUS";
+	private static final String NOT_A_CPC_FILE = "beed7ed4-107c-400f-b0de-0c60abc54344";
+
+	@BeforeAll
+	static void createUnprocessedItem() {
+		given()
+			.multiPart("file", Paths.get("../sample-files/CPCPlus_Success_PreProd.xml").toFile())
+			.when()
+			.post("/");
+	}
+
+	@Test
+	@Tag("acceptance")
+	void testNoSecurityForUnprocessedFiles() {
+		get(CPC_UNPROCESSED_FILES_API_PATH)
+			.then()
+			.statusCode(403);
+	}
+
+	@Test
+	@Tag("acceptance")
```

Comment:
Let's add a meta annotation @AcceptanceTest which has @Test and @Tag("acceptance")